packages/hash/api/src/model/knowledge/page.model.ts

```diff
@@ -411,9 +420,15 @@ export default class extends EntityModel {
       ],
     });
```

Comment:
Doesn't have to be done as part of this PR, but am noticing that the above structural query can be replaced by a single call to `this.getOutgoingLinks(...)`

Reply:
Thanks, you're right. Done in a5b4641ef9a9300ae2a2bb8fe90149633c36075f :)